src/devices/src/virtio/net/persist.rs

```diff
@@ -47,7 +47,7 @@ impl NetConfigSpaceState {
         // v1.1 and older versions do not have optional MAC address.
         warn!("Saving to older snapshot version, optional MAC address will not be saved.");
         match self.guest_mac {
-            Some(mac) => self.guest_mac_old = mac.into(),
```

Comment:
We have a (soft) requirement that each commit should build and pass the tests. If this (and the others) fixes a clippy warning, it should go in the previous commit.